apps/wpcom-block-editor/src/calypso/features/gutenboarding-editor-overrides.js

```diff
@@ -0,0 +1,59 @@
+/* global calypsoifyGutenberg */
+
+/**
+ * External dependencies
+ */
+/* eslint-disable import/no-extraneous-dependencies */
+import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+/* eslint-disable import/no-extraneous-dependencies */
+
+/**
+ * Internal dependencies
+ */
+import './gutenboarding-editor-overrides.scss';
+
+domReady( () => {
+	calypsoifyGutenberg.isGutenboarding && updateEditor();
+	// Hook fallback incase setGutenboardingStatus runs after initial dom render.
+	window.wp.hooks.addAction( 'setGutenboardingStatus', 'a8c-gutenboarding', isGutenboarding => {
+		isGutenboarding && updateEditor();
+	} );
+} );
+
+function updateEditor() {
+	const body = document.querySelector( 'body' );
+	body.classList.add( 'gutenboarding-editor-overrides' );
+
+	updateSettingsBar();
+}
+
+function updateSettingsBar() {
+	const awaitSettingsBar = setInterval( () => {
+		const settingsBar = document.querySelector( '.edit-post-header__settings' );
+		if ( ! settingsBar ) {
+			return;
+		}
+		clearInterval( awaitSettingsBar );
+
+		// 'Update'/'Publish' primary button to become 'Save' tertiary button.
+		const saveButton = settingsBar.querySelector( '.editor-post-publish-button' );
+		saveButton && ( saveButton.innerText = __( 'Save' ) );
+
+		// Create a 'Launch' button.
+		const launchButton = document.createElement( 'button' );
```

Comment:
We end up with `a` _and_ `button`:
```html
<a href="...">
  <button class="gutenboarding-editor-overrides__launch-button components-button is-primary">
    Launch
  </button>
</a>
```

...which isn't a biggie but it has little funky visual hover state (note underline):

![image](https://user-images.githubusercontent.com/87168/73454632-1f172080-4377-11ea-8ffa-9e965d35a3bb.png)

A simple `<a href="..." class="...">Launch</a>` would look normal:

![image](https://user-images.githubusercontent.com/87168/73454670-3950fe80-4377-11ea-8ebd-dd55295f3166.png)

Reply:
#39167 👈